server/sandbox/sandbox.go

```diff
@@ -1,4 +1,4 @@
-package server
+package sandbox
```

Comment:
I was under the impression that libkpod would only handle non-kubernetes related things for the time being, and that pods may be added in later.  If I were to add it to libkpod, should I add it as a subpackage or within the package itself?